libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.repository.ts

```diff
@@ -0,0 +1,101 @@
+import { Injectable } from '@nestjs/common'
+import { logger } from '@island.is/logging'
+import { ParentalLeavePeriod } from './parentalLeavePeriod.model'
+
+// const accessToken = process.env.DIRECTORATE_OF_LABOUR_ACCESS_TOKEN
+
+// TODO implement this class when the endpoints are ready, this is just mocks atm
+
+@Injectable()
+export class DirectorateOfLabourRepository {
+  //   private client: any
+
+  constructor() {
+    logger.debug('Created Directorate of labour repository')
+  }
+
+  //   getClient(): any {
+  //     if (!accessToken) {
+  //       throw new Error(
+  //         'Missing environment variables: DIRECTORATE_OF_LABOUR_ACCESS_TOKEN',
+  //       )
+  //     }
+
+  //     if (this.client) {
+  //       return this.client
+  //     }
+
+  //     return 'new client'
+  //   }
+
+  async getUnions() {
+    return [
+      {
+        id: 'id',
+        name: 'Name',
+      },
+    ]
+  }
+
+  async getPensionFunds() {
```

Comment:
dont you want to add return types to all of these methods?

Reply:
yes, done!